pkg/metrics/ttl/gauge.go

```diff
@@ -16,6 +16,7 @@ import (
 
 var (
 	defaultCleanUpPeriod = 10 * time.Minute
+	DefaultTTL           = 3 * time.Minute
 )
```

Comment:
For snapshots daemons events, it's better to keep the metrics 2 days or longer. It does not occupy so much memory.

Reply:
Yes, I do not use TTL for `snapshots daemons events`. It just works for `FS Metrics`.